content/people/community_roles.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Community Roles"
+date: 2021-06-28T13:44:37-05:00
+draft: false
+layout: page
+aliases: /team/
+---
+TARDIS is developed by a multidisciplinary team. 
+
+The development of TARDIS and associated software tools are made 
+possible through several coordination roles fulfilled by community 
+members. This encompasses a broad scope of responsibilities ranging 
+from direct package development to communication, distribution, and 
+managerial activities. To know about our other members, visit the 
+<a href="../collaboration/">Collaboration</a> page.
+
+## Core Team
+
+TARDIS Core Team is responsible for:
+ - Overseeing the activities and sets the goals for the TARDIS Collaboration
+ - Identifying science opportunities that match TARDIS capabilities or coordinates the addition of capabilities
+ - Identifying education opportunities such as workshops, lecture development, and outreach opportunities
+
+### Members & Roles
+Current TARDIS core team members are as follows (you can click on the role to learn more about it):
+
+<div class ="picture-grid">
+{{< core_members >}}
```

Comment:
Here, my role is listed as "Core contributor" since 2011. Then on the collaboration page I am a "Collaboration coordinator" since 2014. Why are these different? Shouldn't they pull from the same JSON information? The dates are also incorrect in both cases.